forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/BiometricBindingHandler.kt

```diff
@@ -107,4 +107,15 @@ internal class BiometricBindingHandler(titleValue: String,
         }
         return false
     }
+
+    override fun isSupportedBiometricStrong(): Boolean {
+        biometricAuth?.apply {
+            when {
+                this.hasBiometricCapability(BIOMETRIC_STRONG) -> {
```

Comment:
we can simplify this to one line code
 
       return biometricAuth?.hasBiometricCapability(BIOMETRIC_STRONG) ?: false

Reply:
Updated